README.prod.md

```diff
@@ -331,50 +334,42 @@ Maintaining Zulip in production:
 
 * As a measure to mitigate the impact of potential memory leaks in one
   of the Zulip daemons, the service automatically restarts itself
-  every Sunday early morning.  See /etc/cron.d/restart-zulip for the
+  every Sunday early morning.  See `/etc/cron.d/restart-zulip` for the
   precise configuration.
 
 
-=======================================================================
-
-SSO Authentication:
+SSO Authentication
+==================
 
 Zulip supports integrating with a corporate Single-Sign-On solution.
 There are a few ways to do it, but this section documents how to
 configure Zulip to use an SSO solution that best supports Apache and
-will set the REMOTE_USER variable:
+will set the `REMOTE_USER` variable:
 
-(0) Check that /etc/zulip/settings.py has
-"zproject.backends.ZulipRemoteUserBackend" as the only enabled value
-in the "AUTHENTICATION_BACKENDS" list, and that "SSO_APPEND_DOMAIN" is
+(0) Check that `/etc/zulip/settings.py` has
+`zproject.backends.ZulipRemoteUserBackend` as the only enabled value
+in the `AUTHENTICATION_BACKENDS` list, and that `SSO_APPEND_DOMAIN` is
 correct set depending on whether your SSO system uses email addresses
-or just usernames in REMOTE_USER.
+or just usernames in `REMOTE_USER`.
 
 Make sure that you've restarted the Zulip server since making this
 configuration change.
 
-(1) Edit /etc/zulip/zulip.conf and change the puppet_classes line to read:
+(1) Edit `/etc/zulip/zulip.conf` and change the `puppet_classes` line to read:
 
+```
 puppet_classes = zulip::enterprise, zulip::apache_sso
```

Comment:
Just merged a fix to set s/zulip::enterprise/zulip::voyager/ here; will need to fix that merge conflict